include/envoy/server/instance.h

```diff
@@ -179,6 +179,11 @@ class Instance {
    * @return information about the local environment the server is running in.
    */
   virtual const LocalInfo::LocalInfo& localInfo() PURE;
+
+  /**
+   * @return the flush interval of stats sinks.
+   */
+  virtual std::chrono::milliseconds statsFlushInterval() PURE;
```

Comment:
const method, here &  the override

Reply:
The implementation calls statsFlushInterval() of config, which is not a const ref, would you change there too?
https://github.com/envoyproxy/envoy/blob/ed468c9522d6dda82e15c7ba1cc3cb2204ada81b/include/envoy/server/configuration.h#L76